implementation/src/main/java/io/smallrye/config/ConfigMappingGenerator.java

```diff
@@ -352,20 +352,6 @@
         return writer.toByteArray();
     }
 
-    static String generateInterfaceName(final Class<?> classType) {
```

Comment:
Need to move it to prevent getting `ClassNotFoundException` at runtime because `asm` is missing which is wanted as not used and not expected in native mode

Reply:
Interesting. I wonder why we didn't pick this up before.